back-end/routes/taskManagementApi.js

```diff
@@ -3,9 +3,9 @@ const router = express.Router();
 const TaskController = require('../controllers/TaskController');
 
 // Login route
-router.post('/create', TaskController.create);
-router.post('/read', TaskController.read);
-router.post('/update', TaskController.update);
-router.post('/delete', TaskController.delete);
+router.post('/tasks', TaskController.create); // create
+router.get('/tasks', TaskController.read); // read
+router.post('/tasks/:id', TaskController.update); // update
+router.delete('/delete/:id', TaskController.delete); // delete
```

Comment:
can remove this row as well

Reply:
Do we not want to delete the existing task ?

Reply:
HEY